Automattic/woocommerce-payments
# Fix fatal error on sites using WooCommerce Subscription versions below 4.0.0

#### Changes proposed in this Pull Request

Changes introduced in #6344 relies on `WCS_Staging::is_duplicate_site()`. This function was introduced in WCS 4.0.0. Sites using older versions of WCS are encountering a fatal error.

This PR attempts to fix it by using `WC_Subscriptions::is_duplicate_site` for pre 4.0.0 WCS versions. While this prevents the fatal error, it is unclear to me whether this is the correct usage.











#### Testing instructions





* Install and activate WooCommerce Subscriptions 3.1.5 (or any version below 4.0.0)
* Make sure the site is still up and the following fatal error is not thrown

```
Uncaught Error: Call to undefined method WCS_Staging::is_duplicate_site() in /srv/htdocs/wp-content/plugins/woocommerce-payments/includes/subscriptions/class-wc-payments-subscriptions.php:132
```

-------------------


**Post merge**
